selfdrive/car/volkswagen/values.py

```diff
@@ -15,54 +15,49 @@
 
 
 class CarControllerParams:
-  HCA_STEP = 2                   # HCA_01 message frequency 50Hz
-  LDW_STEP = 10                  # LDW_02 message frequency 10Hz
-  GRA_ACC_STEP = 3               # GRA_ACC_01 message frequency 33Hz
+  HCA_STEP = 2                   # HCA_01/HCA_1 message frequency 50Hz
+  GRA_ACC_STEP = 3               # GRA_ACC_01/GRA_Neu message frequency 33Hz
 
-  # Observed documented MQB limits: 3.00 Nm max, rate of change 5.00 Nm/sec.
-  # Limiting rate-of-change based on real-world testing and Comma's safety
-  # requirements for minimum time to lane departure.
+  # Documented lateral limits: 3.00 Nm max, rate of change 5.00 Nm/sec.
+  # MQB vs PQ maximums are shared, but rate-of-change limited differently
+  # in subclasses based on lateral accel testing and safety requirements.
```

Comment:
i'm fine with the subclass if you really prefer it, but consider just passing in carParams like the other brands. I think it's a bit more readable

Reply:
I agree it would be somewhat more clear and readable to fully divide the MQB vs PQ CarControllerParams, and will do so. However, I may revisit subclasses if it makes sense for MLB, MQBevo, or MEB down the line.

Reply:
oh, I meant the opposite. merge them and have an if/else for the MQB/PQ specific stuff

Reply:
The merger would look like reverting ac1f51811840415cd64dda7024a988c74f5c7aaa and then figuring out how to feed a different `STEER_DELTA_UP` for PQ into `apply_std_steer_torque_limits()`. That's what made me split CCP.

It's my preference to do it with two different CCP classes. But I'm not the only one that has to live with the code, so if you see a better path, I'm more than happy to get onboard.

Reply:
see hyundai for example https://github.com/commaai/openpilot/blob/b6910923d18af7332f781a7c83b2601fd7862e3f/selfdrive/car/hyundai/values.py#L24

Reply:
Ah! Yeah, I can do that. Reverting to draft till tomorrow.

Reply:
See if this most recent update feels more readable to you, I like it.